src/styles/index.scss

```diff
@@ -0,0 +1,4 @@
+@import "base/common";
```

Comment:
uhmmm why I need to combine all the styles into one index?

Reply:
It may overkill now, but when a project gets bigger you can separate your styles to keep track of each component's style with ease

Reply:
it's ok for now, as you said when project gets bigger you really want to keep things isolated by concerns (every component with it's test, style, actions?, communication-layer, etc).